packages/sit-onyx/src/components/OnyxDialog/OnyxDialog.vue

```diff
@@ -103,13 +104,18 @@ watch(
     width: max-content;
 
     position: fixed;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
 
     &::backdrop {
       background-color: var(--onyx-color-component-opacity-backdrop);
     }
+    margin: auto;
+    align-self: anchor-center;
+    &--dont-support-anchor {
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      margin: 0;
+    }
 
```

Comment:
we should keep things simple and use an approach that doesn't require the anchor API. There is this cool article, which describes how this can be done: https://levelup.gitconnected.com/dont-use-transform-to-center-element-b378587ad1db


```suggestion
    position: fixed;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
    width: fit-content;
    height: fit-content;
```